configs/dh-defaults.prop

```diff
@@ -45,123 +14,41 @@ measurement.log_and_reset=false
-#
-Comm.fatalErrorHandlerFactoryClass=io.deephaven.console.utils.ProcessEnvironmentRedirectFactory
-
-##### CompilerTools Settings #####
-
-CompilerTools.logEnabledDefault=false
-
 QueryLibrary.defaultPackageImportList=default_package_imports.txt
 QueryLibrary.defaultClassImportList=default_class_imports.txt
 QueryLibrary.defaultStaticImportList=default_static_imports.txt
 
-########## Deephaven Groovy and Python Session defaults ##########
-
-GroovyDeephavenSession.defaultScriptPath=<devroot>
-GroovyDeephavenSession.initScripts=core/deephaven_core_utils.groovy
-
-PythonDeephavenSession.defaultScriptPath=<devroot>
-PythonDeephavenSession.initScripts=core/deephaven_jpy_init.py
-
-WorkerPythonEnvironment.defaultScriptPath=<devroot>
-
-########## Deephaven UpdatePerformanceTracker Properties ##########
-UpdatePerformanceTracker.reportingMode=LISTENER_ONLY
-UpdatePerformanceTracker.reportIntervalMillis=60000
-
-########## Deephaven QueryPerformanceRecorder Properties ##########
-QueryPerformanceRecorder.packageFilter.internal=defaultPackageFilters.qpr
+StringUtils.cacheSize=0
 
-########## Deephaven Local Data Refresh Properties ##########
-TableDataRefreshService.tableLocationsRefreshMillis=10000
-TableDataRefreshService.tableSizeRefreshMillis=1000
+TrackedFileHandleFactory.maxOpenFiles=1024
 
-# Defines the directory for pidfiles
-pidFileDirectory=/etc/deephaven/run
+Calendar.default=USNYSE
+Calendar.importPath=/default_calendar_imports.txt
 
-default.processEnvironmentFactory=io.deephaven.util.process.DefaultProcessEnvironment$Factory
+Plot.theme.default=LIGHT
+Plot.theme.internalPath=<devroot>/configs/chartthemes
 
 columnsFile=DeephavenColumns.xml
 
-TrackedFileHandleFactory.maxOpenFiles=4096
-StringUtils.cacheSize=65536
-
-scriptSessionProviders=Groovy,Python
-scriptSessionExtension.groovy=Groovy
-scriptSessionExtension.py=Python
-scriptSessionProvider.Groovy=io.deephaven.engine.util.GroovyDeephavenSession
-scriptSessionProvider.Python=io.deephaven.engine.util.PythonDeephavenSession
-
-#
-# Web API Server configuration
-#
-Webapi.server.port=8123
-# Full path to an image for Web Login Splash and Settings Logo
-# Recommended 350x350 or smaller
-# Supports png, jpg, and svg
-# Set permissions to 755
-#Webapi.logo.splash=/full/path/logo.png
-#Webapi.logo.settings=/full/path/logo.jpg
-#Webapi.support.contact=test@domain.com
-#Webapi.support.documentation=https://test.domain.com
-# Default heap sizes in GB
-Webapi.console.heap.default=4
-Webapi.persistentquery.heap.default=4
-# A full path to a directory containing JavaScript Plugins
-#Webapi.plugins=/full/path/plugins
-# A comma separate list of names for an App level JavaScript Plugins
-#Webapi.app.plugins=AppPluginName1,AppPluginName2
-# Set to true to display system badge
-#Webapi.user.defaults.systemBadge=false
-
-OneClick.allowedPattern.USym=^[A-Za-z0-9/.]+$
-OneClick.enforceUpperCaseColumns=USym
-
-UpdateGraphProcessor.checkTableOperations=true
```

Comment:
Please put this back.

Reply:
I *think* Colin did a copy/paste and/or rename, but since there were file renames, the diff isn't very good.

Reply:
I think he just moved `grpc-api.prop` to `dh-defaults.prop`. Either way, I would like `UpdateGraphProcessor.checkTableOperations=true` in our production applications.